pyro/distributions/block_naf.py

```diff
@@ -0,0 +1,229 @@
+# This implementation uses some code from https://github.com/nicola-decao/BNAF under the MIT license.
```

Comment:
can you please clarify what this means? we are only able to merge code with certain sorts of licenses

Reply:
I used the details from a few lines of equations from the author's implementation but not any whole blocks of code verbatim: [https://github.com/nicola-decao/BNAF/blob/master/bnaf.py](https://github.com/nicola-decao/BNAF/blob/master/bnaf.py)

Reply:
If you aren't pasting code verbatim, could you change the comment to

> This implementation is adapted from ...

Reply:
Done!